autometa/taxonomy/lca.py

```diff
@@ -94,18 +92,19 @@ class LCA(NCBI):
 
     """
 
-    def __init__(self, dbdir: str, verbose: bool = False):
+    def __init__(self, dbdir: str, verbose: bool = False, outdir: str = None):
```

Comment:
Now pickled database files will be written for each metagenome to their respective output directory. This should resolve #170

Reply:
That would mean it would have to process/pickle the databases for every input?
It might be beneficial to run as a separate process in Nextflow so that [storeDir](https://www.nextflow.io/docs/latest/process.html#storedir) could be used?